### ISSUE TITLE
Errno 18 Invalid cross-device 

### ISSUE THREAD

**Issue by Necosyx** [ranger/ranger#2171]
#### Runtime Environment

ranger version: ranger 1.9.3
Python version: 3.8.6 (default, Sep 30 2020, 04:00:38) [GCC 10.2.0]
Locale: en_US.UTF-8

#### Current Behavior

On folder moves from my arch linux to a external decrypted drive I got this error : 

![IssueRanger](https://user-images.githubusercontent.com/25346179/100597509-d5cc9000-32fd-11eb-8c7c-95fcecd27aa1.png)

#### Expected Behavior

Here, we expect to correctly move the folders into my external drive. 

#### Possible Solutions

Maybe a Shutil functions mistake ? 

#### Steps to reproduce
1. Cut 2 Folders from your linux home directory
2. Paste them to an internal big drive ( NTFS Formatted )
3.  Enjoy..

Anyway, big thanks for your works man ! 

```

**Comment by toonn** [comment]
Hmm, looks like this involves problems with symlinks and veracrypt, is one of the directories in question a symlink?

**Comment by Necosyx** [comment]
No, it shouldn't be a symlinks because I never mount this drive. But actually, the output involves a folder already copied, it's like he cannot override this naughty one

**Comment by toonn** [comment]
And you don't run into any problems copying this same directory using `cp` or `rsync`?

**Comment by Necosyx** [comment]
Indeed, no problems using cp
